### DISCUSSION TITLE
隆重介绍，润太祖真实身份：贪污受贿导致人命事故的，入狱局长公子，曾公子 https://github.com/MarionNoel/juzhanggongzi/discussions

### DISCUSSION THREAD

**Discussion by MarionNoel** [The-Run-Philosophy-Organization/run#547] [Ideas]
https://github.com/MarionNoel/juzhanggongzi/discussions

局长公子呼风唤雨，没句实话。
搁以前拦不住你。矿下“暴民”之死，没影响你润美国，呵呵。

少扯大话。就问入狱局长公子，在美国过得怎么样？

**Comment by MarionNoel** [comment]
他关心扩张他的资产，而不在意耗材的死活。

**Comment by MarionNoel** [comment]
这个官方润学太祖，真是手段高超，难怪这么能敛财

**Comment by tonyqus** [comment]
好奇你是怎么八出这些信息的
